src/celeritas/ext/detail/GeantBremsstrahlungProcess.cc

```diff
@@ -107,7 +107,7 @@ void GeantBremsstrahlungProcess::InitialiseEnergyLossProcess(
         em_model->SetLowEnergyLimit(energy_min);
         em_model->SetHighEnergyLimit(energy_limit);
         em_model->SetSecondaryThreshold(em_parameters->BremsstrahlungTh());
-        em_model->SetLPMFlag(false);
+        G4EmParameters::Instance()->SetLPM(false);
```

Comment:
I think this should be removed: if the model selection is "all", won't this make the LPM flag in the EM parameters false when the relativistic model is added?

Reply:
Okay, I overlooked that [G4SeltzerBergerModel](https://geant4.kek.jp/lxr/ident?v=11.1.2;i=G4SeltzerBergerModel) is now derived from [G4VEmModel](https://geant4.kek.jp/lxr/ident?i=G4VEmModel).